recipes/FaceInpainting.py

```
@@ -26,6 +25,10 @@
 from daras_ai_v2.stable_diffusion import InpaintingModels
 
 
+class HttpUrlHttpUrl:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)